07_deploy_masters.sh

```diff
@@ -37,4 +37,41 @@ done
 for node in $(jq -r .nodes[].name $MASTER_NODES_FILE); do
   openstack baremetal node deploy --config-drive configdrive $node
 done
-# FIXME(shardy) we should wait for the node deploy to complete (or fail)
+
+# Note we have to tolerate failure with the || echo "0" due to this issue
+# https://storyboard.openstack.org/#!/story/2005093
+NUM_ACTIVE=$(openstack baremetal node list --fields name --fields provision_state | grep master | grep active | wc -l || echo 0)
```

Comment:
Hmm the workaround for the failing command isn't working, failing like:

```./07_deploy_masters.sh: line 45: [: 0
0: integer expression expected
```

So I guess I need to switch to a string comparison or ensure the echo 0 is cast to an int